test/expression_test.ts

```diff
@@ -92,4 +92,59 @@ describe('expressions', () => {
     expectTranslate('foo.bar;').to.equal('foo.bar;');
     expectTranslate('foo[bar];').to.equal('foo[bar];');
   });
+  it('should cast numeric values to int when bitwise operators are used for a LHS exp RHS numeric expression',
```

Comment:
isn't this test largely redundant with the test table above? I'd drop it, doesn't add value.